autoscale-api/src/main/java/com/sequenceiq/periscope/doc/ApiDescription.java

```diff
@@ -153,7 +153,11 @@ private ApiDescription() {
     public static class LoadAlertJsonProperties {
         public static final String LOAD_ALERT_CONFIGURATION_MIN_RESOUCE_VALUE = "The lower bound for the resource";
         public static final String LOAD_ALERT_CONFIGURATION_MAX_RESOUCE_VALUE = "The upper bound for the resource";
-        public static final String LOAD_ALERT_CONFIGURATION_COOL_DOWN_MINS_VALUE = "CoolDown between successive cluster autoscaling actions.";
+        public static final String LOAD_ALERT_CONFIGURATION_COOL_DOWN_MINS_VALUE = "CoolDown Mins between successive cluster autoscaling actions.";
```

Comment:
"This is overridden by the scale-up/scale-down cooldown parameters, if they are specified"

Reply:
yes